javascript/README.md

```diff
@@ -57,20 +57,19 @@ Code snippets in Javascript demonstrating how to use various modules of the [Med
 * [API:Backlinks](https://www.mediawiki.org/wiki/API:Backlinks)
   * [get_backlinks.js](get_backlinks.js): list pages which link to a certain page
 * [API:Random](https://www.mediawiki.org/wiki/API:Backlinks)
-  * [get_random.js](get_random.js): get a list of random pages 
+  * [get_random.js](get_random.js): get a list of random pages
 * [API:Alllinks](https://www.mediawiki.org/wiki/API:Alllinks)
   * [get_alllinks.js](get_alllinks.js): list links to a namespace
 * [API:Embeddedin](https://www.mediawiki.org/wiki/API:Embeddedin)
   * [get_embedded_pages.js](get_embedded_pages.js): get all page(s) that embed a page
 * [API:Mergehistory](https://www.mediawiki.org/wiki/API:Mergehistory)
   * [mergehistory.js](mergehistory.js): Merge the page revisions of Oldpage
     dating up to 2015-12-31T04:37:41Z into Newpage
-* [API:Contributors](https://www.mediawiki.org/wiki/API:Contributors)
-  * [get_contributors.js](get_contributors.js): get request to list all logged-in contributors and count of anonymous contributors to a page
-* [API:Stashedit](https://www.mediawiki.org/wiki/API:Stashedit)
-  * [stashedit.js](stashedit.js): Prepare an edit in shared cache
+* [API:FileArchive](https://www.mediawiki.org/wiki/User:Liz_Kariuki/Sandbox/API:Filearchive)
```

Comment:
This should link to the page in the main namespace, i.e. `https://www.mediawiki.org/wiki/API:Filearchive`